docs/Makefile

```diff
@@ -1,3 +1,18 @@
+# Copyright 2018 Intel Corporation
```

Comment:
Much of this Makefile was auto-generated, thus the lack of a Copyright header. The changes would be copyright, but the rest of the file is probably in fair-use land.